src/App.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import {
+  BrowserRouter as Router,
+  Route,
+  Switch,
+} from 'react-router-dom';
+import Home from './containers/home/Home';
+import Details from './containers/detail/Details';
+import List from './containers/list/List';
+
+
+export default class App extends React.Component { 
+  render() {
+    return (
+      <>
+        <div className="main">
+          <Router>
+            <Switch>
+
+              <Route
+                path="/"
+                exact
+                // Can pass props this way
+                // render={(routerProps) => <Home {...routerProps} />}
+                // ------------------------------------------------------------
+                // can't pass prop easier syntax 
```

Comment:
These are the equivalent. 
https://medium.com/@migcoder/difference-between-render-and-component-prop-on-react-router-v4-368ca7fedbec